mindsdb/api/mysql/mysql_proxy/executor/executor_commands.py

```diff
@@ -54,11 +55,13 @@
     DropKnowledgeBase,
     DropMLEngine,
     DropPredictor,
+    DropSkill,
     DropTrigger,
     Evaluate,
     FinetunePredictor,
     RetrainPredictor,
     UpdateChatBot,
+    UpdateSkill
```

Comment:
We need to cut a new release of `mindsdb_sql` to bring in https://github.com/mindsdb/mindsdb_sql/pull/317